docs/events/timeline-ui-overview.asciidoc

```diff
@@ -2,84 +2,72 @@
 Timeline. You can select whether the fields use predefined values or values
-retrieved from the alert. For more information, see <<timeline-templates-ui>>.
+retrieved from the alert. For more information, refer to <<timeline-templates-ui>>.
 
 
 [discrete]
 [[refine-timeline-results]]
 == View and refine Timeline results
 
-Using the drop-down options
-by the KQL bar, you can select whether <<det-engine-terminology, alerts>>,
-other raw events, or both are displayed in the Timeline.
+Using the drop-down options by the KQL bar, you can select whether <<det-engine-terminology, alerts>>, other raw events, or both are displayed in the Timeline.
 
 [discrete]
 [[conf-timeline-display]]
 == Configure Timeline event context and display
 
-Many events in Timeline are presented in easy-to-follow rendered views that
-include relevant contextual information. These views are called
-*Event Renderers*, and they can be configured via the Settings icon in the upper
-left corner of the result's pane:
+Many types of events automatically appear in preconfigured views that provide various kinds of context, called *Event Renderers*. You can see and turn them on or off via the Settings icon in the upper left corner of the results pane:
 
 [role="screenshot"]
 image::images/timeline-ui-renderer.png[]
 
-The example above displays a flow event renderer. If you see a particular item
-that interests you, you can drag it to the dropzone for further investigation.
+The example above displays the Flow event renderer, which highlights the movement of data between its source and destination. If you see a particular part of the rendered event that interests you, you can drag it up to the dropzone for further investigation.
 
-Other display options include:
+You can also modify a Timeline's display in other ways:
 
-* View in full screen mode
-* Add, remove, reorder, and resize columns
-* Add notes to individual events
-* Add investigation notes for the entire Timeline
-* Pin interesting events in the Timeline
+* View it in full screen mode.
+* Add, remove, reorder, or resize columns.
+* Add notes to individual events.
+* Add investigation notes to the entire Timeline.
+* Pin interesting events to the Timeline.
 
 [discrete]
 [[narrow-expand]]
 == Narrow or expand your KQL query
 
-You can specify logical `AND` and `OR` operations with an item's placement in
-the drop area. Multiple horizontal filters use `AND` logic, vertical filters use
-`OR`.
+By placing items within the drop area, you turn them into query filters. Their relative placement specifies their logical relationships: horizontally adjacent filters use `AND`, while vertically adjacent filters use `OR`.
 
 [discrete]
 [[pivot]]
 == Edit existing filters
 
 Click a filter to access additional operations such as exclude, temporarily
-disable, or delete items from the query:
+disable, or delete:
 
 [role="screenshot"]
 image::images/timeline-ui-filter-options.png[width=30%]
```

Comment:
Here's the current image:
<img width="622" alt="timeline-ui-filter-options" src="https://user-images.githubusercontent.com/79124755/156790725-6cf8a908-252c-4332-8031-244a7e368c00.png">